cmd/up/up.go

```diff
@@ -232,6 +232,10 @@ func (up *upContext) start(autoDeploy, build bool) error {
 		return fmt.Errorf("'okteto up' is not allowed in the current namespace")
 	}
 
+	if up.Dev.Docker.Enabled && ns.Annotations[model.OktetoURLAnnotation] == okteto.CloudURL {
+		return fmt.Errorf("Docker support requires privileged containers and it's not supported in Okteto Cloud")
```

Comment:
do you think it's important to talk about privileged containers? Maybe we just mention that this feature is not available on Cloud?

Reply:
Also, what's going to happen on OE if privileged containers are not enabled? do we handle that well?

Reply:
You get the standard error, something like this:
```
 x  couldn't activate your development container
    pods "api-57879cc69c-" is forbidden: PodSecurityPolicy: unable to admit pod: [spec.containers[1].securityContext.privileged: Invalid value: true: Privileged containers are not allowed]
```

Reply:
could catch that error for DinD and show something more user-friendly? (since it's not the user the one that's adding the privileged container).

Reply:
which message would you show?